deployment/pipecat-cloud/rest-reference/openapi-start.json

```diff
@@ -95,6 +95,18 @@
             "description": "Optional configuration for the Daily meeting token. Only used when createDailyRoom is true.\n\nSee [Daily API documentation](https://docs.daily.co/reference/rest-api/meeting-tokens/config) for supported properties.",
             "example": { "is_owner": true, "enable_auto_recording": true }
           },
+          "enableDefaultIceServers": {
+            "type": "boolean",
+            "default": false,
+            "description": "Whether to use the default STUN/TURN ICE servers provided by Pipecat Cloud.",
+            "example": true
+          },
+          "transport": {
+            "type": "string",
+            "description": "Explicitly specify which transport the bot should use.",
+            "enum": ["daily", "webrtc", "twilio", "telnyx", "plivo", "exotel"],
+            "example": "webrtc"
+          },
```

Comment:
This property requires the changes from [this PR](https://github.com/daily-co/pipecat-cloud-sandbox/pull/322).